packages/cli/src/commands/setup/deploy/providers/aws-serverless.js

```diff
@@ -24,19 +24,37 @@ custom:
 
 provider:
   name: aws
-  runtime: nodejs12.x
+  runtime: nodejs14.x
   region: us-east-2 # This is the AWS region where the service will be deployed.
   httpApi: # HTTP API is used by default. To learn about the available options in API Gateway, see https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-vs-rest.html
-    cors: true
+    cors:
+      allowOrigins:
+        - '*' # This is the default value. You can remove this line if you want to restrict the CORS to a specific origin.
+      allowCredentials: true
+      allowedHeaders:
+        - authorization
+        - auth-provider
+        - content-Type
+        - X-Amz-Date
+        - X-Api-Key
+        - X-Amz-Security-Token
+        - X-Amz-User-Agent
     payload: '1.0'
+    useProviderTags: true # https://www.serverless.com/framework/docs/deprecations/#AWS_HTTP_API_USE_PROVIDER_TAGS
   stackTags: # Add CloudFormation stack tags here
     source: serverless
     name: Redwood Lambda API with HTTP API Gateway
   tags: # Add service wide tags here
     name: Redwood Lambda API with HTTP API Gateway
+  lambdaHashingVersion: 20201221 # https://www.serverless.com/framework/docs/deprecations/#LAMBDA_HASHING_VERSION_V2
 
 package:
   individually: true
+  patterns:
```

Comment:
@virtuoushub could you give me a clue what this is required for? Everything else in this PR looks good.

Reply:
@dac09 - they stem from reading through @thedavidprice's comment here: https://github.com/redwoodjs/redwood/issues/3284#issuecomment-922375958 and more specifically here as that first one is a rather large info dump: https://gist.github.com/thedavidprice/ac67656b1a3e2b32f31799cd4fb7cbec#gistcomment-3897679

I have not test this theory, but it is very possible #3782 does most of this cleanup for us already?

see also:
- https://www.prisma.io/docs/guides/deployment/deployment-guides/deploying-to-aws-lambda#package-pattern-in-serverlessyml
- https://gist.github.com/thedavidprice/ac67656b1a3e2b32f31799cd4fb7cbec#gistcomment-3897679

Reply:
I see makes sense. Let me take it for a spin, and will merge in if it deploys all ok!